src/openrct2/libopenrct2.vcxproj

```diff
@@ -886,6 +887,7 @@
     <ClCompile Include="paint\tile_element\Paint.Surface.cpp" />
     <ClCompile Include="paint\tile_element\Paint.TileElement.cpp" />
     <ClCompile Include="paint\tile_element\Paint.Wall.cpp" />
+    <ClCompile Include="paint\tile_element\Tunnel.cpp" />
```

Comment:
For consistency and because this file actually contains paint functions, could you rename this to `Paint.Tunnel.cpp`? IIRC, this was originally done because of compilation conflicts between files with the same name in MSVC.